src/screens/MainApp/DiagnoseRequest/screens/ImageReview/index.js

```diff
@@ -44,18 +61,23 @@ const ImageVisualization = (props) => {
       const currentImages = deleteActiveIndexImage()
 
       if (currentImages.length === 0) {
+        props.navigation.popToTop()
         NavigationService.navigate('NoPhotoDisclaimer')
-      } else {
-        if (activeIndex > 0) {
-          setActiveIndex(activeIndex - 1)
-        }
+      } else if (activeIndex > 0) {
+        setActiveIndex(activeIndex - 1)
       }
     }
 
-    const changeImage = () => {
-      const changedImageArray = images.slice()
-      changedImageArray[activeIndex] = newImages[activeIndex]
-      setImages(changedImageArray)
+    const changeImage = async () => {
+      try {
+        const newImage = (await ImageService.openCamera())[0]
```

Comment:
🤔 this looks ugly. What if we use array destructuring to extract the first element?
```javascript
const [ newImage ] = await ImageService.openCamera()
```